src/proj7BittingCerratoCohenEllmer/bantam/Main.java

```diff
@@ -39,6 +40,9 @@ public static void main(String[] args) {
                         System.out.println(
                             "*** " + filename + " had " 
                                 + errorCount + " errors! ***");
+                        for (Error error : bantamErrorHandler.getErrorList()) {
+                            System.out.println(error);
+                        }
```

Comment:
This was useful in debugging but we might not want to keep it in our submission since Dale didnt ask for it.